### ISSUE TITLE
Velero has to be configured even when dr module is not installed

### ISSUE THREAD

**Issue by FedericoAntoniazzi** [sighupio/fury-distribution#207]
Hello there,

I see the schema validation fails when `spec.distribution.modules.dr.type` is set to `none` because it still requires `spec.distribution.modules.dr.velero` to be set.

```yaml
apiVersion: kfd.sighup.io/v1alpha2
kind: OnPremises
metadata:
  name: furydemo
spec:
  distributionVersion: v1.27.3
  ...
  distribution:
    modules:
      dr:
        type: none
      ...
```

```console
❯ furyctl validate config
INFO Downloading distribution...                  
INFO Compatibility patches applied for v1.27.3    
ERRO error while validating against schema: jsonschema: '/spec/distribution/modules/dr' does not validate with file:///var/folders/mf/vblq1rpx01571403sbmyb6m40000gn/T/furyctl-2325438801/data/schemas/public/onpremises-kfd-v1alpha2.json#/properties/spec/$ref/properties/distribution/$ref/properties/modules/$ref/properties/dr/$ref/required: missing properties: 'velero' 
ERRO configuration file validation failed
```

## Additional info
```console
❯ furyctl version
buildTime: 2024-03-26T12:02:24Z
gitCommit: 334082c40678df83615e40a14376a540e6243a1c
goVersion: go1.22.0
osArch: amd64
version: 0.27.8
```

**Comment by nutellinoit** [comment]
as a workaround you just need to add `velero: {}`

we should fix the schema rules

**Comment by ralgozino** [comment]
seems to have been fixed in #119 

can't reproduce in latest versions:

```bash
❯ furyctl version
DEBU logging to: /Users/ramiroalgozino/src/test/furyctl-on-multipass/.furyctl/furyctl.1724857568-58432.log
buildTime: 2024-08-28T14:10:43Z
gitCommit: c3347ff7f99cd12167d55171abada31a893c18b6
goVersion: go1.22.3
osArch: arm64
version: 0.29.5-rc.3
❯ yq '.spec.distribution.modules.dr' furyctl.yaml
type: none

```
